sovtoken/sovtoken/client_authnr.py

```diff
@@ -53,10 +53,16 @@ def authenticate(self, req_data, identifier: str = None,
     # The verkey will be unobtainable from input, it will raise CouldNotAuthenticate
     # Raises UnknownIdentifier if input is not a valid base58 value
     def authenticate_xfer(self, req_data, verifier):
+        extra_fields = []
+        if req_data[OPERATION].get(EXTRA, None):
+            extra_fields.append(req_data[OPERATION][EXTRA])
+        if req_data.get(f.TAA_ACCEPTANCE.nm, None):
+            extra_fields.append(req_data[f.TAA_ACCEPTANCE.nm])
         return self.verify_signtures_on_payments(req_data[OPERATION][INPUTS],
```

Comment:
Please add/extend unit tests for `verify_signtures_on_payments` taking into account extra and TAA fields